launcher/recreate

```diff
@@ -21,7 +24,16 @@ fi
 
 if podman inspect --type container $CONTAINER_NAME &>/dev/null; then
   podman stop $CONTAINER_NAME || podman stop $CONTAINER_NAME
-  toolbox rm $CONTAINER_NAME
+
+  if [[ "$(uname)" == "Darwin" ]]; then
+   podman rm $CONTAINER_NAME
+  else
+   toolbox rm $CONTAINER_NAME
+  fi
 fi
 
-toolbox create $CONTAINER_NAME --image $CONTAINER_IMAGE
+if [[ "$(uname)" == "Darwin" ]]; then
```

Comment:
could you integrate something like this, in a common file, and use `if [[ "$HAS_TOOLBOX" == y ]]; then`
```
if toolbox --help > /dev/null 2>/dev/null;
then
  HAS_TOOLBOX=y
else
  HAS_TOOLBOX=y
fi
```